tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -765,11 +769,12 @@ def fit(self,
             refit_full=kwargs['refit_full'],
             set_best_to_refit_full=kwargs['set_best_to_refit_full'],
             save_space=kwargs['save_space'],
+            calibrate=calibrate
```

Comment:
just pass `kwargs['calibrate']`, as is done with the other arguments here

Reply:
pulled calibrate from `kwargs`